Tests/SwiftLintFrameworkTests/BaselineTests.swift

```diff
@@ -0,0 +1,182 @@
+@testable import SwiftLintBuiltInRules
+@testable import SwiftLintCore
+import XCTest
+
+private var temporaryFilePath: String {
+    let result = URL(
+        fileURLWithPath: NSTemporaryDirectory(),
+        isDirectory: true
+    ).appendingPathComponent(UUID().uuidString).path
+
+#if os(macOS)
+    return "/private" + result
+#else
+    return result
+#endif
+}
+
+private var sourceFilePath: String = {
+    temporaryFilePath + ".swift"
+}()
+
+final class BaselineTests: XCTestCase {
+    private var violations: [StyleViolation] {
+        [
+            ArrayInitRule.description,
+            BlockBasedKVORule.description,
+            ClosingBraceRule.description,
+            DirectReturnRule.description
+        ].violations
+    }
+
+    private var baseline: Baseline {
+        Baseline(violations: violations)
+    }
+
+    func testWritingAndReading() throws {
+        try testBlock {
```

Comment:
Is `testBlock` needed here?

Reply:
So it will pass without it, but unless the sample code is on disk, `Baseline(violations: Self.violations)` will be unable to read the file to obtain the lines of text - it will go with `""` in these cases, but `File.swift` in SourceKittenFramework will print to stderr:

```
Could not read contents of `/private/var/folders/7h/f16htcls4l391mnfhbw21628y1r4t3/T/B5A0ACEE-62F4-40B6-B22E-96E7BCF2D75F.swift`
Could not read contents of `/private/var/folders/7h/f16htcls4l391mnfhbw21628y1r4t3/T/B5A0ACEE-62F4-40B6-B22E-96E7BCF2D75F.swift`
Could not read contents of `/private/var/folders/7h/f16htcls4l391mnfhbw21628y1r4t3/T/B5A0ACEE-62F4-40B6-B22E-96E7BCF2D75F.swift`
Could not read contents of `/private/var/folders/7h/f16htcls4l391mnfhbw21628y1r4t3/T/B5A0ACEE-62F4-40B6-B22E-96E7BCF2D75F.swift
```